content/docs/guides/mediawiki-tabular-data.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Comparison with MediaWiki Tabular Data
+sidebar:
+  order: 3
+---
+
+<table>
+  <tr>
+    <th>Authors</th>
+    <td>Jakob Voß</td>
+  </tr>
+</table>
+
+[MediaWiki](https://www.mediawiki.org/) is the software used to run Wikipedia and related projects of the Wikimedia Foundation, including the media file repository [Wikimedia Commons](https://commons.wikimedia.org/). Commons hosts mostly images but also some records with tabular data. The [MediaWiki Tabular Data Model](https://www.mediawiki.org/wiki/Help:Tabular_data) was inspired by Data Package version 1 but it slightly differs from current Data Package specification, as described below.
+
+## Property Comparison
+
+A [MediaWiki tabular data page](https://www.mediawiki.org/wiki/Help:Tabular_data) describes and contains an individual table of data similar to a [Data Resource](/standard/data-resource/) with inline tabular data. Both are serialized as JSON objects, but the former comes as a page with unique name in a MediaWiki instance (such as Wikimedia Commons).
+
+### Top-level Properties
+
+MediaWiki Tabular Data has three required and two optional top-level properties. Most of these properties map to corresponding properties of a Data Resource:
+
+| MediaWiki Tabular Data                                                                                                                     | Data Package Table Schema                                                             |
```

Comment:
For easier maintenance, I would use the most simple form of a Markdown table (here and below), without whitespace or repeated `------` to align:

```
| MediaWiki Tabular Data | Data Package Table Schema |
| --- | --- |
| - (implied by page name) | [name](/standard/data-resource/#name) (required) is a string |
```

Reply:
Yes, I wrote it that way but the linter had another opinion.

Reply:
Ow, that's an annoying linter setting. Ping @roll do contributors have to write Markdown tables that verbosely?

Reply:
Yea we need to fix it. I had the same problem with CSVW doc so I just added it to the `.prettierignore`